Monika After Story/game/script-songs.rpy

```diff
@@ -1462,6 +1462,31 @@ label mas_song_wonderwall_lyrics:
     m 1hubsb "{i}~You're my wonderwall~{/i}"
     return
 
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_songs_database,
+            eventlabel="mas_song_tanabata_song",
+            category=[store.mas_songs.TYPE_SHORT],
+            prompt=('Tanabata song'),
+            random=False,
+            conditional="seen_event('monika_tanabata')"
+        ),
+        code="SNG"
+    )
+
+label mas_song_tanabata_song:
+    m 1dua "{i}The bamboo leaves rustle,{/i}"
```

Comment:
```suggestion
    m 1dua "{i}The bamboo leaves rustle,{/i}{w=0.5}{nw}"
```
I'd use alternating extends here. Having just the second half of a sentence displayed doesn't look good.

Also I believe it has been brought up that, when using song translations, the tempo should fit the original song. This translation is pretty literal and doesn't really fit it imo. The syllable counts are all over the place.
Though it might be better to get @multimokia 's opinion here, I'm no expert on music.

Reply:
Tempo doesn't necessarily have to match. But if a song is genuinely slow, we should ideally at least loosely try to match